specs/phase0/beacon-chain.md

```diff
@@ -1317,7 +1317,9 @@ def get_attestation_deltas(state: BeaconState) -> Tuple[Sequence[Gwei], Sequence
         attesting_balance = get_total_balance(state, unslashed_attesting_indices)
         for index in eligible_validator_indices:
             if index in unslashed_attesting_indices:
-                rewards[index] += get_base_reward(state, index) * attesting_balance // total_balance
+                increment = EFFECTIVE_BALANCE_INCREMENT  # Factored out from balance totals to avoid uint64 overflow
```

Comment:
Maybe also note that the current total active balance upper bound is around ~10B ETH here or in `get_total_active_balance` function?

Reply:
added in `get_total_balance`